.github/workflows/validate-devfiles-minikube.yaml

```diff
@@ -37,3 +37,15 @@ jobs:
         odo: "latest"
     - name: Validate the devfile stacks
       run: tests/test.sh odo
+  check_architectures:
+    name: Check Stack and Sample Architectures
+    continue-on-error: true
```

Comment:
Do we need this set here and down below?

Reply:
updated